llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -16172,8 +16233,6 @@ static SDValue performSETCCCombine(SDNode *N, SelectionDAG &DAG,
       N0.getConstantOperandVal(1) != UINT64_C(0xffffffff))
     return SDValue();
 
-  // Looking for an equality compare.
-  ISD::CondCode Cond = cast<CondCodeSDNode>(N->getOperand(2))->get();
   if (!isIntEqualitySetCC(Cond))
     return SDValue();
```

Comment:
Nit, you could pull up this early exit to line 16217 since both combineVectorSizedSetCCEquality and the existing combine need it